### PR TITLE
feat: add index.html with RapiPDF to openapiv2

### PR DESCRIPTION
**Pull request by odsod**
For quickly generating a PDF document.

### CODE REVIEW

File: pdf/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+  <head>
+    <script src="https://cdn.jsdelivr.net/gh/mrin9/RapiPdf@master/dist/rapipdf-min.js"></script>
+  </head>
+  <body>
+    <rapi-pdf
+      spec-url="openapiv2/book.swagger.yaml"
```
Comment (radhus) [inline pdf/index.html:8]: Note that this particular file does not exist in the repo yet 😬

File: pdf/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      button-bg="#121212"
```
Comment (jenseo) [inline pdf/index.html:9]: To make it compatible in both dark and light mode, we could use theme colors here (theme.colors...). But we can adjust to that later, when we see it works as expected!

**Review by jenseo — APPROVED** [review]

**Review by radhus — APPROVED** [review]

File: pdf/index.html (hunk 2 above)
Comment (odsod) [reply pdf/index.html:9]: Yeah!

File: pdf/index.html (hunk 1 above)
Comment (odsod) [reply pdf/index.html:8]: But soon... :innocent:

**Pull request merged**